specification/compute/resource-manager/Microsoft.Compute/stable/2021-08-01/disk.json

```diff
@@ -3482,7 +3485,17 @@
         },
         "completionPercent": {
           "type": "number",
-          "description": "Percentage complete for the background copy when a resource is created via the CopyStart operation."
+          "description": "Percentage complete for the background copy of disk restore point when source resource is from a different region."
```

Comment:
completionPercent reflects copy state for both snapshot and restore point right?